stock_picking_package_info/models/stock.py

```diff
@@ -27,8 +31,34 @@ def _catch_operations(self):
         self.packages = [
             operation.result_package_id.id for operation in
             self.pack_operation_ids if operation.result_package_id]
+        self._calculate_package_info()
         self._calculate_package_totals()
 
+    def _calculate_package_info(self):
```

Comment:
Esto se puede hacer seguro, porque yo lo he hecho con un one2many, pero bueno, apúntalo si no como lista de to-do, para no retrasar más esto.